lib/src/input/figma/entities/layers/component.dart

```diff
@@ -85,9 +81,13 @@ class Component extends FigmaFrame implements AbstractFigmaNodeFactory {
 
   @override
   FigmaNode createFigmaNode(Map<String, dynamic> json) {
-    var component = Component.fromJson(json);
+    var component = Component.fromJson(json)
+      ..figmaStyleProperty = FigmaNode.figmaStylePropertyFromJson(json);
 
-    component.style = StyleExtractor().getStyle(json);
+    // component.style = StyleExtractor().getStyle(json);
+
+    /// IF figmaStyleProperty already have the fills
+    /// do we still need a fills field?
```

Comment:
Ideally, we do not need the fills property if we already give it to the FigmaStyle 👍🏾